astropy/modeling/tests/test_models_quantities.py

```diff
@@ -562,6 +562,7 @@
     PowerLaw1D,
     ExponentialCutoffPowerLaw1D,
     BrokenPowerLaw1D,
+    Voigt1D,
```

Comment:
The one thing that was surprising was that `Voigt1D` using the `wofz` method is unstable for the general fitting problem using the `TRFLSQFitter`, and the `LMLSQFitter` while the `humlicek2` method is not. This maybe related to #13986.

I need to investigate the fitting problem being tested here anyways.

Reply:
But at the time it seems we were only using `LevMarLSQFitter` in this test.

Reply:
Could not reproduce this with either scipy 1.9.3 or 1.10.0.dev, although by setting a more distant initial centre like `'x_0': 520 * u.nm` I can force `LMLSQFitter` (using `wofz`, but not `humlicek2`), but not `TRFLSQFitter` to fail similarly – apparently running out of bounds with too large `x_0`, huge `fwhm_L` and negative `fwhm_G` (the latter of which is pathological in itself; I think there may be some extra curating required for the accepted inputs to `scipy.special.wofz`).
It's also not quite clear to me what is actually being fitted here – would 
```python
            x = np.linspace(1, 3, 100) * model['evaluation'][0][0].unit
            y = np.exp(-x.value ** 2) * model['evaluation'][0][1].unit
```
not just create a Gaussian (wing) from 1-3 micron in this case?

Reply:
Apparently it does not really test the fit, just the borderline behaviour of the fitter: https://github.com/astropy/astropy/pull/6183#discussion_r121764766

Reply:
Yeah the test is mostly to test that the fitter "runs" with the model.

Reply:
Scipy results just seem to be architecture or compile-flag dependent – my mambaforge installations of 1.9.3 are failing in `LMLSQFitter` as well, even with the default parameters (the above test was with the 1.10.0dev I built myself on arm64).